runtime/src/lib.rs

```diff
@@ -212,7 +211,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("logion"),
 	impl_name: create_runtime_str!("logion"),
 	authoring_version: 1,
-	spec_version: 3,
+	spec_version: 4,
```

Comment:
Shall we apply semver spec_versions? If yes, spec version should be 3000 (for 0.3.0 -> 000_003_000 i.e. 3000).